core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -363,10 +366,13 @@ protected Node(final Environment environment, Collection<Class<? extends Plugin>
                     .flatMap(p -> p.getNamedWriteables().stream()))
                 .flatMap(Function.identity()).collect(Collectors.toList());
             final NamedWriteableRegistry namedWriteableRegistry = new NamedWriteableRegistry(namedWriteables);
+            NamedXContentRegistry xContentRegistry = new NamedXContentRegistry(Stream.of(
+                    searchModule.getNamedXContents().stream()
```

Comment:
I use this funny looking construct to match with the constructs above. It'll eventually have more than one module it has to stream, just like above.